src/pages/home/home.spec.ts

```diff
@@ -5,13 +5,29 @@ import { Subject } from 'rxjs';
 import { TestUtils } from '../../test';
 
 import { AddressBookProvider } from '../../providers/address-book/address-book';
+import { ExternalLinkProvider } from '../../providers/external-link/external-link';
+import { PersistenceProvider } from '../../providers/persistence/persistence';
+import { AddPage } from '../add/add';
+import { CopayersPage } from '../add/copayers/copayers';
+import { WalletDetailsPage } from '../wallet-details/wallet-details';
 import { ConfigProvider } from './../../providers/config/config';
+import { ActivityPage } from './activity/activity';
 import { HomePage } from './home';
+import { ProposalsPage } from './proposals/proposals';
 
-describe('HomePage', () => {
+fdescribe('HomePage', () => {
```

Comment:
Looks like an `fdescribe` was left in here

Reply:
fixed up